nixneovim.nix

```diff
@@ -238,6 +246,7 @@ in
         {
           programs.neovim = {
             enable = true;
+            defaultEditor = cfg.defaultEditor;
```

Comment:
This only sets the default for home-manager.
Please also add this to the non-home-manager module

Reply:
Added